### ISSUE TITLE
Variable Learning Rate Matrix

### ISSUE THREAD

**Issue by githubaccount11** [pytorch/pytorch#33681]

**Comment by githubaccount11** [comment]
Was this empty?
